src/napari/layers/shapes/shapes.py

```diff
@@ -2747,43 +2747,52 @@ def _update_thumbnail(self, event=None):
 
             self.thumbnail = colormapped
 
-    def remove_selected(self) -> None:
-        """Remove any selected shapes."""
-        index = list(self.selected_data)
-        to_remove = sorted(index, reverse=True)
```

Comment:
@brisvag same question here, the original implementation of `remove_selected` had `sorted`. So were you asking to just move the sorting from the new `remove` to the re-implemented `remove_selected`?